kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/CreateTopicRequest.java

```diff
@@ -38,24 +41,32 @@
   @JsonProperty("replication_factor")
   public abstract Optional<Short> getReplicationFactor();
 
+  @JsonProperty("replicas_assignments")
+  public abstract Map<Integer, List<Integer>> getReplicasAssignments();
+
   @JsonProperty("configs")
   public abstract ImmutableList<ConfigEntry> getConfigs();
 
   public static Builder builder() {
-    return new AutoValue_CreateTopicRequest.Builder();
+    return new AutoValue_CreateTopicRequest.Builder()
+        .setReplicasAssignments(Collections.emptyMap());
```

Comment:
Do we need to default?

Reply:
Yes, unfortunately - otherwise the `AutoValue` validation complains that a field that's neither `@Nullable`, nor `Optional` has _not_ been initialized.